r/1. Getting Started/Getting Started.Rmd

```diff
@@ -0,0 +1,185 @@
+
+
+### Process discovery
+
+To proceed, it's important to grasp the available built-in processes of openEO. We've already utilized a few of these processes in our earlier queries, like `list_collection_ids` and `describe_collection`.
+
+```{r}
+# list all the available processes
+list_processes()
+```
+
+```{r}
+describe_process("aggregate_temporal")
+
+```
+
+Find more information on these processes in [this page](https://processes.openeo.org/).
+
+
+## Data access
+
+A common task in EO is applying a formula to several spectral bands to compute an ‘index’, such as NDVI, NDWI, EVI, etc. In this tutorial, we’ll go through the steps to extract EVI (enhanced vegetation index) values and timeseries and discuss some openEO concepts along the way.
+
+To calculate the EVI, we need red, blue, and (near) infrared spectral components. These spectral bands are part of the well-known Sentinel-2 data set and are available on the current backend under collection ID `SENTINEL2_L2A`. So, let's load this collection.
+
+```{r}
+p <- processes()
+datacube <- p$load_collection(
+    id = "SENTINEL2_L2A",
+    spatial_extent=list(west = 5.14, south = 51.17, east = 5.17, north = 51.19),
+    temporal_extent=list("2021-02-01", "2021-04-30"),
+    bands=list("B08","B04","B02")
+)
+
+```
+
+Here, we use the `load_collection` process that loads a collection from the current backend using its ID. It loads the collection as a datacube restricted by spatial_extent, temporal_extent, bands, and properties.
+
+Additionally, by filtering as early as possible (directly in `load_collection()` in this case), we ensure the backend only loads the data we are interested in for better performance and to keep the processing costs low. In this example, we filter the data based on the spatial extent, temporal extent, and bands.
+
+Furthermore, in this example, we implemented bbox for `spatial_extent`; however, user can import their spatial files and feed them into the process as a feature collection.
+
+
+## Data processing: Calculate EVI
+
+While openEO offers a built-in process for calculating NDVI(ndvi()), this capability hasn't yet been implemented for EVI or other indices. Instead, openEO provides support for most other indices through an auxiliary subpackage called Awesome Spectral Indices. However, users also have the option to perform band math independently, as demonstrated in this notebook. The choice between the two methods depends on user preference.
+
+In the following cell, we create the EVI calculation and store it as a process graph on the openEO service.
+
+Most of the processes offered by openEO services are standardized, which means that it is possible to use mathematical operators like `+` and `-` and similar coherently between different services. That also allowed us to overload the primitive mathematical operators in R to make it easy to use.
+ 
+
+```{r}
+evi_cube <- p$reduce_dimension(data = data, dimension = "bands",reducer = function(data,context) {
```

Comment:
data = data 
should be data = datacube
shouldn't it?

Reply:
it should be 😄 thank you for reminder